AntennaCharacteristics.ipynb

```diff
@@ -0,0 +1,601 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# Directivity and S-parameters computation of patch antenna \n",
+    "\n",
+    "A rectangular patch antenna is a type of microstrip antenna consisting of a rectangular conductive patch placed on a dielectric substrate with a ground plane on the opposite side. These antennas are widely used in wireless communication applications due to their simple design, ease of fabrication, and low profile.\n",
+    "\n",
+    "<img src=\"img/PatchAntenna.png\" alt=\"MIM\"  width=\"400\"/>\n",
+    "\n",
+    "In this notebook, we will demonstrate how to use Tidy3D to simulate a rectangular patch antenna [1] and compute key performance metrics. These include S-parameters using the [TerminalComponentModeler](https://docs.flexcompute.com/projects/tidy3d/en/v2.7.3/api/_autosummary/tidy3d.plugins.smatrix.TerminalComponentModeler.html), as well as directivity, axial ratio, and polarized far-field components using the [DirectivityMonitor](https://docs.flexcompute.com/projects/tidy3d/en/pre-2.8/api/_autosummary/tidy3d.DirectivityMonitor.html).\n"
```

Comment:
I think we should link to the latest version in the docs, so

[https://docs.flexcompute.com/projects/tidy3d/en/latest/api/_autosummary/tidy3d.plugins.smatrix.TerminalComponentModeler.html](https://docs.flexcompute.com/projects/tidy3d/en/latest/api/_autosummary/tidy3d.plugins.smatrix.TerminalComponentModeler.html)